web/test/data/public-project.json

```diff
@@ -0,0 +1,6 @@
+{
+  "type": "public",
+  "title": "Demo public project",
+  "description": "Demo public project description added",
+  "url": "https://www.pipelinedashboard.io"
+}
```

Comment:
I was initially thinking of a complete project, but I think this is better, it tests that the application can handle minimum data. I will create new tickets for more details data like repositories etc

Reply:
okay